get_instagram.py

```diff
@@ -0,0 +1,21 @@
+from InstagramAPI import InstagramAPI
+from key import *
+
+
+def user():
+	api = InstagramAPI(username=user_name, password=password)
```

Comment:
How many spaces are there?

Reply:
4

Reply:
It's not true.
`def ` - it's 4 symbols. Your code started after this.